### DISCUSSION TITLE
api for retriving best selling products from etsy

### DISCUSSION THREAD

**Discussion by Imdad-cr7** [etsy/open-api#941] [Q&A] [answered]
I am currently working on etsy api and i am stuck with some issues i am trying to access data through api for products such as best selling or trending products but i couldn't find anything can anyone help me out with it and let me know does etsy provide such api for trending products or best selling products.

**Comment by Gareth064** [comment, marked as answer]
The API doesn't offer any info like that I'm afraid.

From a quick glance, on my phone, you can get all active listings and then from those you can see how many Favourites they have had.

Not exactly good enough for trending etc.

**Comment by showtroylove** [comment]
Regrettably the answer to your questions is NO.  We discussed this or similar on this thread #930
